### PR TITLE
Added a collectable coin

### PR DESCRIPTION
**Pull request by KevinDwo**
Added another event and the player rectangle is now initialized before the game runs.

If the player reaches the coin, it wont be drawn.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -13,9 +13,17 @@
 
 x = windowWidth / 2
 y = windowHeight / 2
+
+# initialize player
+player = pygame.Rect(x, y, 40, 40)
+
+# intitialize coin
+coin = pygame.Rect(100, 100, 16, 16)
+coinCollected = True
```
Comment (Ludwig-Work) [inline main.py:22]: The `coinCollected` variable currently is `True` if the coin has *not* been collected, and `False` if it *has* been collected. Perhaps this should be swapped, considering the variable name?

File: main.py [hunk 2]
```diff
@@ -35,10 +43,20 @@
...
+    # change background color
     window.fill((0, 0, 0))
-    pygame.draw.rect(window, (0, 0, 255), (x, y, 40, 40))
+
+    # draw coin
+    if coinCollected:
+        pygame.draw.circle(window, (212, 175, 55), (100, 100), 16)
+
+    # draw player
+    player = pygame.Rect(x, y, 40, 40)
```
Comment (Ludwig-Work) [inline main.py:58]: I don't see the advantage of initializing the player object before the loop, when it is overridden here in the first iteration of the loop anyway. Effectively, the player coordinates are now stored twice seperately (in the `player` object and in the variables `x` and `y`), so I'm not sure if the `player` object brings any advantage.

File: main.py [hunk 3]
```diff
@@ -13,9 +13,17 @@
 
 x = windowWidth / 2
 y = windowHeight / 2
+
+# initialize player
+player = pygame.Rect(x, y, 40, 40)
+
+# intitialize coin
+coin = pygame.Rect(100, 100, 16, 16)
```
Comment (Ludwig-Uni) [inline main.py:21]: You initialize a `coin` object as a rectangle here, but never use it later. I suggest to drop this.
The actual coin that is drawn later is a circle and not a rectangle anyway.

**Review by Ludwig-Work — CHANGES_REQUESTED** [review]
I pointed out a couple things in the comments. Most importantly, the `coinCollected` state is a little confusing. The `player` object is probably fine, we could perhaps drop the `x` and `y` variables later.

**Pull request closed without merging**